topics/developer-experience/index.md

```diff
@@ -0,0 +1,8 @@
+---
+display_name: Developer experience
+related: dx
```

Comment:
You don't need to copy the entire file, we have a mechanism for this! Please convert this into an ["alias"](https://github.com/github/explore/blob/main/docs/API.md#aliases) and delete the other file.

Reply:
You're right! Thank you for clarifying it, it's done now.